libtest/Makefile.am

```diff
@@ -21,6 +21,8 @@ libtest_libsyslog_ng_test_a_SOURCES =   \
 	libtest/persist_lib.c		\
 	libtest/mock-transport.c	\
 	libtest/mock-transport.h	\
+	libtest/mock-cfg-parser.c	\
```

Comment:
This should probably added to the `CMakeLists.txt` as well.

Reply:
Done.